GraphNeuralNetworks/docs/src_tutorials/introductory_tutorials/traffic_prediction.jl

```diff
@@ -0,0 +1,134 @@
+# # Traffic Prediction using recurrent Temporal Graph Convolutional Network
+
+# In this tutorial, we will learn how to use a recurrent Temporal Graph Convolutional Network (TGCN) to predict traffic in a spatio-temporal setting. Traffic forecasting is the problem of predicting future traffic trends on a road network given historical traffic data, such as, in our case, traffic speed and time of day.
+
+# ## Import
+
+# We start by importing the necessary libraries. We use `GraphNeuralNetworks.jl`, `Flux.jl` and `MLDatasets.jl`, among others.
+
+using Flux, GraphNeuralNetworks
+using Flux.Losses: mae
+using MLDatasets: METRLA
+using Statistics, Plots
+
+# ## Dataset: METR-LA
+
+# We use the `METR-LA` dataset from the paper [Diffusion Convolutional Recurrent Neural Network: Data-driven Traffic Forecasting](https://arxiv.org/pdf/1707.01926.pdf), which contains traffic data from loop detectors in the highway of Los Angeles County. The dataset contains traffic speed data from March 1, 2012 to June 30, 2012. The data is collected every 5 minutes, resulting in 12 observations per hour, from 207 sensors. Each sensor is a node in the graph, and the edges represent the distances between the sensors.
```

Comment:
What does it mean `the edges represent the distances between the sensors`? should be clarified

Reply:
better something like: the edge weights are the distances between the sensor locations?